internal/application/command.go

```diff
@@ -128,7 +129,8 @@ func readDeviceResource(device models.Device, resourceName string, attributes st
 	}
 
 	// convert CommandValue to Event
-	res, edgexErr = transformer.CommandValuesToEventDTO(results, device.Name, dr.Name, dic)
+	configuration := container.ConfigurationFrom(dic.Get)
+	res, edgexErr = transformer.CommandValuesToEventDTO(results, device.Name, dr.Name, configuration.Device.DataTransform, dic)
```

Comment:
I am confused on why readDeviceResource is being changed for PR that is about Set.
Is this change for `DataTransform` in scope for what this PR is for? If so, why?

Reply:
OK, thanks for the explination.